### PR TITLE
Update JWT docs

### PR DESCRIPTION
**Pull request by volcanonoodle**
Update https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt
Add that:
* Grafana does not support refresh tokens
* Claim validation through the `expect_claims` config option only supports exact matching. Pattern matching is **not** supported.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md [hunk 1]
```diff
@@ -183,7 +187,8 @@ key_id = my-key-id
 
 By default, only `"exp"`, `"nbf"` and `"iat"` claims are validated.
 
-You might also want to validate that other claims are really what you expect them to be.
+You might also want to validate that other claims are really what you expect them to be, which you can do by setting the `expect_claims` configuration option.
```
Comment (irenerl24) [inline docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md:190]: Consider validating that other claims match your expectations by using the `expect_claims` configuration option.

File: docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md [hunk 2]
```diff
@@ -183,7 +187,8 @@ key_id = my-key-id
...
+Token claims will need to match the values set here exactly.
```
Comment (irenerl24) [inline docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md:191]: Token claims must match exactly the values set here.

Comment (irenerl24) [inline docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md:191]: Token claims must match exactly the values set here.

**Review by kalleep — APPROVED** [review]

**Comment by irenerl24** [comment]
> @irenerl24 could you please take another look when you have some time? Wondering if it looks as intended now 🙏

looks perfect, thank you! :)

**Review by irenerl24 — APPROVED** [review]

**Pull request merged**